activerecord/lib/active_record/model_schema.rb

```diff
@@ -431,6 +434,34 @@ def attribute_types # :nodoc:
         @attribute_types ||= Hash.new(Type.default_value)
       end
 
+      ##
+      # :method: before_load_schema
+      #
+      # :call-seq: before_load_schema(*args, &block)
+      #
+      # Registers a callback to be called on the record class before the schema is loaded successfully. See
+      # ActiveRecord::ModelSchema for more information.
+      def before_load_schema(*, &block)
```

Comment:
Is there really a use case for `before_load_schema` ?

Reply:
Not that I could think of. Now that it's been discussed, I'll remove it in my next commit.